modules/ossm-control-plane-cli.adoc

```diff
@@ -4,7 +4,7 @@ This module is included in the following assemblies:
 ////
 :_content-type: PROCEDURE
 [id="ossm-control-plane-deploy-cli_{context}"]
-= Deploying the control plane from the CLI
+= Deploying the {SMProductShortName} control plane from the CLI
```

Comment:
Service Mesh only supports a single CLI tool, OC.

Reply:
That might not be obvious to a novice user. Also, the OpenShift docs reference various CLI tools. 
Writers often say "by using the CLI" assuming the customer knows which one they mean, when that might not be the case. This is analogous to saying "you can update the <something> by using the browser" instead of saying "by using the {product-title} web console."  
It's better to be precise.

Reply:
Do you really think that novice users can't figure out which CLI we're using when having oc installed is listed as a prerequisite for each of these procedures and all of the commands start with oc?  That we also must include the name of the command line tool in the heading?   The procedure is precise, the headings should be concise.

Reply:
That's not what I meant. We are in agreement. I was mentioning a more general issue.

Reply:
@JStickler and @rolfedh Thanks for the discussion on this. I've updated these instances to "...using the CLI"